counterpoll/main.py

```diff
@@ -209,6 +247,7 @@ def show():
     queue_info = configdb.get_entry('FLEX_COUNTER_TABLE', 'QUEUE')
     port_info = configdb.get_entry('FLEX_COUNTER_TABLE', 'PORT')
     port_drop_info = configdb.get_entry('FLEX_COUNTER_TABLE', PORT_BUFFER_DROP)
+    pg_drop_info = configdb.get_entry('FLEX_COUNTER_TABLE', PG_DROP)
```

Comment:
@ayurkiv-nvda please move it after `pg_wm_info = configdb.get_entry('FLEX_COUNTER_TABLE', 'PG_WATERMARK')`

Reply:
Done